distro/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN GOOS=windows go build -ldflags '-H=windowsgui' -o bin/wsl-gvproxy.exe ./cmd/
     GOOS=linux CGO_ENABLED=0 go build -ldflags '-s -w' -o bin/wsl-vm ./cmd/vm && \
```

Comment:
In the `gvisor-tap-vsock` repo I maintain there is an additional comment to explain why `-ldflags '-H=windowsgui'` was added; as it allows backgrounding of the process. This got lost in the import.

Reply:
added as comment